packages/augur-sdk/src/constants.ts

```diff
@@ -29,15 +31,14 @@ export enum SubscriptionEventNames {
   MarketFinalized = "MarketFinalized",
   MarketMigrated = "MarketMigrated",
   MarketState = "MarketState",
+  NewBlock = "NewBlock",
```

Comment:
cool, was wondering how the UI was going to get new block events. all consistent, sweet!